### ISSUE TITLE
Anything wrong with the config file???

### ISSUE THREAD

**Issue by yang-yk** [HaydenFaulkner/pytorch.repmet#6]
When I train the model using the repmet_flowers_1_12_4.yaml file without any change, the training loss is more than 2500 and the accuracy is about 1%. Can you tell me why? Is there anything should be change? Thank you very much ang hope to receive your answer!

**Comment by qa8306202** [comment]
Me too



Epoch 139/149
----------
Avg Training Loss: 3557.9821 Acc: 0.0167
Avg Validation Loss: 2541.5430 Acc: 0.0033

**Comment by yang-yk** [comment]
> Me too
> Epoch 139/149
> Avg Training Loss: 3557.9821 Acc: 0.0167
> Avg Validation Loss: 2541.5430 Acc: 0.0033

I have solved this problem. It is caused by the pytorch version. If you change the pytorch version to 1.0, it will be okay!

**Comment by qa8306202** [comment]
thank you!

> > Me too
> > Epoch 139/149
> > Avg Training Loss: 3557.9821 Acc: 0.0167
> > Avg Validation Loss: 2541.5430 Acc: 0.0033
> 
> I have solved this problem. It is caused by the pytorch version. If you change the pytorch version to 1.0, it will be okay!

thank u

**Comment by duynn912** [comment]
Hi both,
Has anyone successfully implement the detection part? If so, would you please share your implementation?

**Comment by yang-yk** [comment]
Please detail the implementation of the repmet. Thanks very much!!!

**Comment by duynn912** [comment]
hi @yang-yk,
following the author, the detection may not working, so I wonder any of you has successfully re-implement repmet for the detection task

**Comment by yang-yk** [comment]
> hi @yang-yk,
> following the author, the detection may not working, so I wonder any of you has successfully re-implement repmet for the detection task

hello,
I find the authors of the paper have opened the code in the website https://github.com/jshtok/RepMet. But the code is based on the mxnet framework. There is something wrong with me because I'm not very familiar with the framework. I think we can understand the code and find how it is implemented on the detection tasks together.

**Comment by duynn912** [comment]
Hi @yang-yk ,
Thanks for another implementation. So, I didn't try your mentioned code https://github.com/jshtok/RepMet, I don't know how this work. I only know that this repository https://github.com/HaydenFaulkner/pytorch.repmet is not working with detection task.

**Comment by yang-yk** [comment]
> Hi @yang-yk ,
> Thanks for another implementation. So, I didn't try your mentioned code https://github.com/jshtok/RepMet, I don't know how this work. I only know that this repository https://github.com/HaydenFaulkner/pytorch.repmet is not working with detection task.

Hi @duynn912,
I have tried the detection code of this repository https://github.com/HaydenFaulkner/pytorch.repmet and I found it just was standard faster rcnn. This is not the official code of the paper.  The code of https://github.com/jshtok/RepMet is official but I still have some problems to fully understand it.

**Comment by duynn912** [comment]
Hi @yang-yk,
Thank you for your information!
To me, this repository https://github.com/HaydenFaulkner/pytorch.repmet is identical to your understand. So I also wanna try this official code to see how it work first, then maybe we can have discussion later!

**Comment by yang-yk** [comment]
> Hi @yang-yk,
> Thank you for your information!
> To me, this repository https://github.com/HaydenFaulkner/pytorch.repmet is identical to your understand. So I also wanna try this official code to see how it work first, then maybe we can have discussion later!

Good luck and hope to talk with you again!
